templates/webserver/webserver-deployment.yaml

```diff
@@ -76,6 +76,19 @@ spec:
           env:
           {{- include "standard_airflow_environment" . | indent 10 }}
       containers:
+        {{- if .Values.authSidecar.enabled }}
+        - name: nginx
+          image: {{ .Values.authSidecar.image }}
+          imagePullPolicy: IfNotPresent
+          ports:
+          - containerPort: 8084
+            name: nginx-ui
+            protocol: TCP
```

Comment:
We probably want a livenessprobe, no?

Reply:
And readiness too.

Reply:
Great catch. I made an issue to ensure that all our containers have these https://github.com/astronomer/issues/issues/3108

Reply:
added health check for authsidecar. should it be static or we need it in a configurable way.

Reply:
I think static should be fine in this case. Timeout and failureThreshold seem on the high side to me, but this works.